addons/web/static/src/js/views/kanban/kanban_renderer.js

```diff
@@ -306,6 +306,12 @@ var KanbanRenderer = BasicRenderer.extend({
         var ghostDefs = [];
         for (var $ghost, i = 0; i < nbDivs; i++) {
             $ghost = $('<div>').addClass('o_kanban_record o_kanban_ghost');
+            var recordView = $(fragment).find(".o_kanban_record:not(.o_kanban_ghost)");
+            // When applied record width inline from view, we need to change ghost record
+            // width based on record width.
+            if (recordView.length) {
+                $ghost.css("width", recordView.eq(0).width() + "px");
```

Comment:
Using `width()` will not return the value you want. Kanban records have an inline-style width but their real width (which `width()` will return) will be increased or lowered as `flex` is used. You have to get the inline style, not the computed style.

Note: also, even if you wanted the computed style, you should have used `.outerWidth()` to include borders and paddings.